analysis_options.yaml

```diff
@@ -7,24 +7,19 @@
 # See the configuration guide for more
 # https://github.com/dart-lang/sdk/tree/master/pkg/analyzer#configuring-the-analyzer
 #
-# There are four similar analysis options files in the flutter repos:
+# There are three similar analysis options files in the flutter repos:
 #   - analysis_options.yaml (this file)
-#   - analysis_options_repo.yaml
 #   - packages/flutter/lib/analysis_options_user.yaml
 #   - https://github.com/flutter/plugins/blob/master/analysis_options.yaml
```

Comment:
we should probably change this back to four and point to https://github.com/flutter/engine/blob/master/analysis_options.yaml while we're at it...

Reply:
will do